qiskit/visualization/circuit/matplotlib.py

```diff
@@ -1171,9 +1206,9 @@ def _condition(self, node, node_data, wire_map, cond_xy, glob_data):
         qubit_b = min(node_data[node].q_xy, key=lambda xy: xy[1])
         clbit_b = min(xy_plot, key=lambda xy: xy[1])
 
-        # For IfElseOp, WhileLoopOp or SwitchCaseOp, place the condition
-        # at almost the left edge of the box
-        if isinstance(node.op, (IfElseOp, WhileLoopOp, SwitchCaseOp)):
+        # For IfElseOp, WhileLoopOp or SwitchCaseOp, place the condition line
+        # near the left edge of the box
+        if isinstance(node.op, ControlFlowOp):
             qubit_b = (qubit_b[0], qubit_b[1] - (0.5 * HIG + 0.14))
```

Comment:
This has the effect of bringing `ForLoopOp` into the `if` as well.  That doesn't seem quite right, since `ForLoopOp` doesn't have a condition in the same way.  Was it intentional?

Reply:
Done in https://github.com/Qiskit/qiskit/pull/10842/commits/e1d7982f3177d70010e4c1e264d6c9fecd6aedc5.